packages/react-interactive-tab/src/components/InteractiveTab/renderInteractiveTab.tsx

```diff
@@ -0,0 +1,30 @@
+/** @jsx createElement */
```

Comment:
lets use react v17 features ( we are using v17 ) - aligns also with core 

https://github.com/microsoft/fluentui/pull/32838/files#diff-07b5fa9a3e830a5d704eea58256b26d63c6af7536119dbd16c969536ef3505a9R25